ironfish/src/rpc/routes/chain/getBlocks.ts

```diff
@@ -0,0 +1,132 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import * as yup from 'yup'
+import { Assert } from '../../../assert'
+import { getBlockSize, getTransactionSize } from '../../../network/utils/serializers'
+import { FullNode } from '../../../node'
+import { GENESIS_BLOCK_SEQUENCE } from '../../../primitives/block'
+import { BufferUtils } from '../../../utils'
+import { RpcNotFoundError, RpcValidationError } from '../../adapters'
+import { ApiNamespace } from '../namespaces'
+import { routes } from '../router'
+import { RpcBlock, RpcBlockSchema, serializeRpcBlockHeader } from '../types'
+import { RpcTransaction } from './types'
+
+const MAX_BLOCKS_RANGE = 30
+
+export type GetBlocksRequest = {
+  /**
+   * The starting block height (inclusive).
+   */
+  start: number
+  /**
+   * The ending block height (exclusive).
+   */
+  end: number
+}
+
+export type GetBlocksResponse = {
+  blocks: RpcBlock[]
+}
+
+export const GetBlocksRequestSchema: yup.ObjectSchema<GetBlocksRequest> = yup
+  .object()
+  .shape({
+    start: yup.number(),
+    end: yup.number(),
+  })
+  .defined()
+
+export const GetBlocksResponseSchema: yup.ObjectSchema<GetBlocksResponse> = yup
+  .object({
+    blocks: yup.array(RpcBlockSchema).defined(),
+  })
+  .defined()
+
+routes.register<typeof GetBlocksRequestSchema, GetBlocksResponse>(
+  `${ApiNamespace.chain}/getBlocks`,
```

Comment:
Could you add a test file for this, since the other RPC routes in this folder have them? You should be able to copy and modify the tests in `getBlock.test.ts`, since it's almost the same route as those